src/plugins/flexibleLayout/plugin.js

```diff
@@ -37,7 +37,8 @@ export default function plugin() {
       initialize: function (domainObject) {
         domainObject.configuration = {
           containers: [new Container(50), new Container(50)],
-          rowsLayout: false
+          rowsLayout: false,
+          objectStyles: {}
```

Comment:
See previous comments. Unfortunately we are doing it in some places already, let's try to avoid making the problem worse.

Reply:
Still needs to be addressed.